sktime_dl/deeplearning/tests/test_accuracy.py

```diff
@@ -0,0 +1,112 @@
+'''
+Compare accuracy of the classifiers against results published at
+https://github.com/hfawaz/dl-4-tsc/blob/master/README.md
+Test that accuracy is higher than the published accuracy minus two 
+standard deviations (also published) on the ItalyPowerDemand dataset.
+'''
+
+import pytest
+from flaky import flaky
+import numpy as np
+
+from sktime.datasets import load_italy_power_demand 
+
+from sktime_dl.deeplearning import CNNClassifier
+from sktime_dl.deeplearning import EncoderClassifier
+from sktime_dl.deeplearning import FCNClassifier
+from sktime_dl.deeplearning import MCDCNNClassifier
+from sktime_dl.deeplearning import MCNNClassifier
+from sktime_dl.deeplearning import MLPClassifier
+from sktime_dl.deeplearning import ResNetClassifier
+from sktime_dl.deeplearning import TLENETClassifier
+from sktime_dl.deeplearning import TWIESNClassifier
+from sktime_dl.deeplearning import InceptionTimeClassifier
+
+
+def is_not_value_error(err, *args):
+    return not issubclass(err[0], ValueError)
+
+
+def accuracy_test(network=CNNClassifier(), lower=0.94, upper=1.0):
+    '''
+    Test the classifier accuracy against expected lower and upper bounds.
+    '''
+    print("Start accuracy_test:", network.__class__.__name__)
+
+    X_train, y_train = load_italy_power_demand(split='TRAIN', return_X_y=True)
+    X_test, y_test = load_italy_power_demand(split='TEST', return_X_y=True)
+
+    __ = network.fit(X_train, y_train)
+
+    accuracy = network.score(X_test, y_test)
+    print(network.__class__.__name__, "accuracy:", accuracy)
+    assert(accuracy>lower and accuracy<=upper)
+
+
+@pytest.mark.slow
```

Comment:
Ah okay, flaky now makes sense ...

Reply:
Yes it kicks in occasionally. Keras [uses flaky](https://github.com/keras-team/keras/blob/1cf5218edb23e575a827ca4d849f1d52d21b4bb0/tests/keras/metrics_functional_test.py) too.

Reply:
Yeah that's decent. Presumably useful also if doing e.g. nightly/weekly tests outside of specific pushes/prs to run the slow version